cinch/roles/jenkins_master/defaults/main.yml

```diff
@@ -127,16 +127,13 @@ jenkins_handler_max: 100
 jenkins_handler_idle: 20
 # The cannonical name to use as server_name in nginx configuration file.
 service_name: "{{ hostvars[inventory_hostname]['ansible_host'] | default(inventory_hostname) }}"
-# This is a list of ports to allow in firewall. You should not change this
-# list when running Ansible on already deployed system, since the newly
-# added ports would not be added properly to iptables.
-# TODO: switch to firewalld to eliminate this problem.
+# This is a list of ports with protocol to allow in firewalld.
 firewall_tcp_ports:
```

Comment:
Now that we're on firewalld and the "tcp" part is specified in the items in this list, we might consider changing this to ``firewall_ports`` instead of ``firewall_tcp_ports``.